setup.sh

```diff
@@ -8,13 +8,18 @@ if [[ -z "${LEAPP_ROLES}" ]]; then
     exit
 fi
 
-# Install the xcode Command Line Tools, if they are not installed
-xcode-select --install
-
 # Install Homebrew if not installed
+# This may optionally install the Xcode CLT if it is not already installed.
 which -s brew
 if [[ $? != 0 ]] ; then
-    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" </dev/null
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
The pipe to `/dev/null` was actually preventing the prompt for the user's password to show up when installing homebrew, so I removed that.